.changeset/minor-bottle-phone.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-studio': patch
+---
+
+**BREAKING CHANGE:** Service registration improvements including owner validation and open service UI upon successful registration.
```

Comment:
First off, you put `patch` for `BREAKING CHANGE` :). But regardless, I don't think this UI change is something we consider breaking change in general.

Reply:
We need to include this because the config property `url` is changing to `executionUrl`

Reply:
for breaking change, please do `major`.